tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServiceWrapperCommand.java

```diff
@@ -39,6 +39,22 @@ public ServiceWrapperCommand( IServer server )
         _server = server;
     }
 
+    public String[] getServiceWrapper() throws Exception
+    {
+
+        if( _server == null )
+        {
+            return getStaticServiceWrapper();
+        }
+        else
+        {
+            String[] wrappers = getDynamicServiceWrapper();
+            updateServiceWrapperStaticFile( wrappers );
```

Comment:
For update static file we have another ticket to do this. So can you just review these code except this one. I will fix this issue on another ticket.